lightcurve/urls.py

```diff
@@ -8,6 +8,7 @@
     url(r'^(?P<lightcurve_id>[0-9]+)/save_observation_default$', 'save_observation_default', name='save_observation_default'),
     url(r'^(?P<lightcurve_id>[0-9]+)/add_image_toggle$', 'add_image_toggle', name='add_image_toggle'),
     url(r'^(?P<lightcurve_id>[0-9]+)/edit_lightcurve_name$', 'edit_lightcurve_name', name='edit_lightcurve_name'),
+    url(r'^(?P<lightcurve_id>[0-9]+)/download_file$', 'download_file', name='download_file'),
```

Comment:
Very minor nit, but could we just make this `/download$`?  I think `download_file` is redundant.

Reply:
Done